conf/defaults.ini

```diff
@@ -130,7 +130,25 @@ logging = false
 # This setting also applies to core backend HTTP data sources where query requests use an HTTP client with timeout set.
 timeout = 30
 
-# If enabled and user is not anonymous, data proxy will add X-Grafana-User header with username into the request, default is false.
+# How long the data proxy waits before sending a keepalive request. Default is 30 seconds.
+keep_alive_seconds = 30
+
+# How long the data proxy waits for a successful TLS Handshake before timing out. Default is 90 seconds.
```

Comment:
Discussed this in the team, and we figured we might as well avoid double-stating the default in the comments. This one is also out of sync with the actual value.
```suggestion
# How many seconds the data proxy waits for a successful TLS Handshake before timing out.
```